WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -298,14 +298,21 @@ extension OrderListViewController {
     func startListeningToNotifications() {
         let nc = NotificationCenter.default
         nc.addObserver(self, selector: #selector(defaultAccountWasUpdated), name: .defaultAccountWasUpdated, object: nil)
+        nc.addObserver(self, selector: #selector(defaultSiteWasUpdated), name: .StoresManagerDidUpdateDefaultSite, object: nil)
```

Comment:
⚠️ When https://github.com/woocommerce/woocommerce-ios/pull/2850 is merged, it doesn't look like we will need this at all since the Orders tab and other VCs will be recreated anyway.

Reply:
yes! originally I wasn't sure if the tab bar controller recreation PRs (https://github.com/woocommerce/woocommerce-ios/pull/2850 and https://github.com/woocommerce/woocommerce-ios/pull/2886) would be merged this sprint, and wanted to make the crash fix https://github.com/woocommerce/woocommerce-ios/pull/2879 along with this PR to make sure orders tab are only showing orders with the current site. now that the tab bar controller PRs were merged, no need for these changes anymore! removed in https://github.com/woocommerce/woocommerce-ios/pull/2897/commits/d6f8daf8ff384469d850a857ee718cfc090877ad